lib/rspec/core/option_parser.rb

```diff
@@ -143,11 +144,15 @@ def parser(options)
 
   **** Filtering/tags ****
 
-    In addition to the following options for selecting specific files, groups,
-    or examples, you can select a single example by appending the line number to
+    In addition to the following options for selecting specific files, groups, or
+    examples, you can select individual examples by appending the line number(s) to
     the filename:
 
-      rspec path/to/a_spec.rb:37
+      rspec path/to/a_spec.rb:37:87
+
+    You can also pass example ids enclosed in square brackets:
+
+      rspec path/to/a_spec.rb[1:5,1:6] # run the 5th and 6th examples/groups defined in the 1st group
```

Comment:
I think this is a good compromise. I can definitely figure it out myself, and can edit it if it changes while refactoring.